### PR TITLE
integrate jib to build docker image

### LINKED ISSUE #50: dockerization

### PR DESCRIPTION
**Pull request by bellatoris**
제곧내

fix #50

### CODE REVIEW

**Comment by bellatoris** [comment]
Current dependencies on/for this PR:
* main
  * **PR #55** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/55" target="_blank">Graphite</a> 
    * **PR #56** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/56" target="_blank">Graphite</a> 
      * **PR #57** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/57" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/57?utm_source=stack-comment).

File: scripts/run-java.sh [hunk 1]
```diff
@@ -0,0 +1,654 @@
+#!/bin/sh
```
Comment (Zeniuus) [inline scripts/run-java.sh:1]: 출처 적으면 좋을 듯

File: subprojects/packaging/build.gradle.kts [hunk 2]
```diff
@@ -18,4 +21,42 @@ dependencies {
...
+        credHelper.helper = "ecr-login"
+        tags = setOf("latest")
+    }
+    container {
+        entrypoint = listOf("./app/run-java.sh")
+        environment = mapOf(
+            "JAVA_MAIN_CLASS" to "club.staircrusher.packaging.SccServerApplicationKt",
+            "JAVA_LIB_DIR" to "/app/libs/*:/app/classes:/app/resources",
+        )
+        mainClass = "club.staircrusher.packaging.SccServerApplicationKt"
```
Comment (Zeniuus) [inline subprojects/packaging/build.gradle.kts:58]: 얘랑 JAVA_MAIN_CLASS 둘 다 넣어줘야 하나요?

**Review by Zeniuus — APPROVED** [review]
jib 만세~

File: subprojects/packaging/build.gradle.kts (hunk 2 above)
Comment (bellatoris) [reply subprojects/packaging/build.gradle.kts:58]: 아뇽 `JAVA_MAIN_CLASS` 만 있으면 돼요

File: scripts/run-java.sh (hunk 1 above)
Comment (bellatoris) [reply scripts/run-java.sh:1]: 넹 이후의 PR에서 수정해놓을게요

**Comment by bellatoris** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/57?utm_source=stack-comment).


Job Reference: GPrmpXx1RQXYQgyKqvrw

File: subprojects/packaging/build.gradle.kts (hunk 2 above)
Comment (bellatoris) [reply subprojects/packaging/build.gradle.kts:58]: 근데 그냥 내비둘게요 ㅋㅋ

**Pull request merged**